withdraw.js

```diff
@@ -0,0 +1,45 @@
+const withdrawalForm = document.getElementById("withdrawal-form");
+const withdrawalAmountElem = document.getElementById("withdrawal-amount");
+const accountPinElem = document.getElementById("account-pin");
+
+const currentUser = getUserByAccountNumber(currentUserAccountNumber);
+
+const withdraw = () => {
+  // get the previous transaction before this one
+  const currentBalance = getUserCurrentBalance();
+  const allUsers = getAllUsers();
+
+  if (accountPinElem.value !== currentUser.accountPin) {
+    alert("Incorrect PIN.");
+    return;
+  }
+  if (withdrawalAmountElem.value > currentBalance) {
+    alert("Insufficient Balance");
+    return;
+  }
+  if (withdrawalAmountElem.value < 0) {
```

Comment:
This check should be (withdrawalAmountElem.value < 1). You can't withdraw 0 Naira.

Reply:
Done